Lib/test/test_dis.py

```diff
@@ -1050,12 +1105,12 @@ def test_explicit_first_line(self):
         self.assertEqual(list(actual), expected_opinfo_outer)
 
     def test_source_line_in_disassembly(self):
-        # Use the line in the source code
-        actual = dis.Bytecode(simple).dis()[:3]
+        # Use the line in the source code (split extracts the line no)
+        actual = dis.Bytecode(simple).dis().split(" ")[0]
```

Comment:
Since we only want the first entry, `.partition(" ")[0]` would be a better option here (it doesn't really matter all that much, given that disassembly lines are short, but we may as well avoid pointlessly splitting the rest of the line up into fields when we don't care about them).